be/src/olap/storage_engine.cpp

```diff
@@ -918,95 +918,36 @@ OLAPStatus StorageEngine::add_tablet(TTabletId tablet_id, SchemaHash schema_hash
 OLAPStatus StorageEngine::add_transaction(
```

Comment:
transaction and txn is the same. Using one variable name may be better.

Reply:
transaction related method in storage engine will be removed in the future. They will be replaced with create_xxx_task. Txn related method will not be called outside.